common-service-core/src/main/java/org/zowe/apiml/util/CorsUtils.java

```diff
@@ -34,7 +34,8 @@ public class CorsUtils {
     static {
         allowedCorsHttpMethods = Collections.unmodifiableList(Arrays.asList(
             HttpMethod.GET.name(), HttpMethod.HEAD.name(), HttpMethod.POST.name(),
-            HttpMethod.DELETE.name(), HttpMethod.PUT.name(), HttpMethod.OPTIONS.name()
+            HttpMethod.PATCH.name(), HttpMethod.DELETE.name(), HttpMethod.PUT.name(),
+            HttpMethod.OPTIONS.name()
```

Comment:
I think it would be good to have them also as a configuration option together with the rest of the CORS parameters